app/lib/server/methods/addUsersToRoom.js

```diff
@@ -30,7 +30,7 @@ Meteor.methods({
 		const userInRoom = subscription != null;
 
 		// Can't add to direct room ever
-		if (room.t === 'd') {
+		if (room.t === 'd') { // TODO CHANGE
```

Comment:
Missing?

Reply:
its for a future feature :x

Reply:
Which future feature? Or you describe what is expected there or you just forgot your reminder there, no one else will understand this.